model/schema/bulkload.yml

```diff
@@ -0,0 +1,198 @@
+id: https://github.com/alliance-genome/agr_curation_schema/src/schema/bulkload.yaml
+
+imports:
+  - core
+
+classes:
+
+  BulkLoadGroup:
+    mixins:
+      - AuditedObject
+    description: This class is use to group together bulk load_files
+    slots:
+      - name
+      - loads
+
+  BulkLoad:
+    mixins:
+      - AuditedObject
+    description: Base class for all loads
+    slots:
+      - name
+      - status
+      - error_message
+      - backend_bulk_load_type
+      - ontology_type
+      - group
+      - load_files
+
+  BulkLoadFile:
+    mixins:
+      - AuditedObject
+    description: This class is used to hold version of the files being loaded
+    slots:
+      - status
+      - md5_sum
+      - local_file_path
+      - file_size
+      - s3_path
+      - s3_url
+      - record_count
+      - error_message
+      - bulk_load
+
+  BulkScheduledLoad:
+    is_a: BulkLoad
+    description: This a bulk load that includes a cron scheuld
+    slots:
+      - schedule_active
+      - cron_schedule
+      - scheduling_error_message
+
+  BulkFMSLoad:
+    is_a: BulkScheduledLoad
+    description: This bulk load automatically pulls files from the BulkFMSLoad
+    slots:
+      - data_type
+      - data_sub_type
+
+  BulkURLLoad:
+    is_a: BulkScheduledLoad
+    description: This bulk load automatically pulls files from a defined BulkURLLoad
+    slots:
+      - url
+
+  BulkManualLoad:
+    is_a: BulkLoad
+    description: This bulk load is used by DQM's to submit their files to the curation system
+    slots:
+      - backend_load_type
+
+slots:
+
+  loads:
+    multivalued: true
+    range: BulkLoad
+
+  status:
+    description: Status used to capture the progress of the load
+    range: bulk_load_status_enum
+
+  error_message:
+    description: Error message string if an error occurs
+    range: string
+
+  backend_bulk_load_type:
+    range: backend_bulk_load_type_enum
+
+  ontology_type:
+    range: ontology_bulk_load_type_enum
+
+  group:
```

Comment:
'bulk_load_group'? Again, to avoid generic sounding slot names that might want to be instantiated later in a broader context.

Reply:
Changed to bulkload_group